### PR TITLE
Centered collapsible with a fixed width

### PR DESCRIPTION
**Pull request by jithinlal**

### CODE REVIEW

**Comment by daltonfury42** [comment]
Please keep updating the same PR. It's a bit difficult to review when you keep opening new PRs.

**Comment by maaverik** [comment]
It crashes for me too when I click the button. Also, styling has to be done later on. @jithinlal, could you merge the latest code and try to recreate the problem?

Screenshot from 2020-08-24 00-47-25

Screenshot from 2020-08-24 00-49-46

**Comment by maaverik** [comment]
@daltonfury42  could you take a look at this when you get some time? It's something to do with the joinQueuHandler being passed incorrectly to the joinForm component. I looked at it for a while but I can't figure out what's wrong.

**Comment by maaverik** [comment]
Got it. I was looking at the wrong place. In /components/pages/Admin/index.jsx, in addNewItem, nothing's being returned. This function is being passed into the joinQueueHandler in Form.jsx. Adding a return there fixes it.

  const addNewItem = (name, contactNumber) => {
```
    return TokenService.create(name, contactNumber, false, queueId)
      .then((response) => {
        setItems([
```

File: simplq/src/components/common/Collapsible/Collapsible.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const StandardCollapsible = (props) => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <Grid container direction="column" alignItems="center" justify="center">
+      <Grid item cs={6}>
+        <ListItem button onClick={() => setOpen(!open)}>
+          <ListItemIcon>
+            <AddIcon />
```
Comment (daltonfury42) [inline simplq/src/components/common/Collapsible/Collapsible.jsx:18]: Can we also pass down the icon as a prop?

**Review by daltonfury42 — CHANGES_REQUESTED** [review]

**Comment by jithinlal** [comment]
Hi,
Is this PR still open?

**Comment by daltonfury42** [comment]
Yes

**Comment by maaverik** [comment]
Closing because this page has been redesigned.

**Pull request closed without merging**